virttest/utils_netperf.py

```diff
@@ -197,25 +192,17 @@ def __init__(self, address, netperf_path, md5sum="", netperf_source="",
         :param install: Whether need install netperf or not.
         """
         self.client = client
-        if client == "nc":
-            self.prompt = r"^\w:\\.*>\s*$"
-            self.linesep = "\r\n"
-            self.status_test_command = "echo %errorlevel%"
-        else:
-            self.prompt = "^\[.*\][\#\$]\s*$"
-            self.linesep = "\n"
-            self.status_test_command = "echo $?"
 
         self.package = NetperfPackage(address, netperf_path, md5sum,
                                       netperf_source, client, port, username,
-                                      password)
+                                      password, prompt, linesep, status_test_command)
         self.netserver_path, self.netperf_path = self.package.install(install,
                                                                       compile_option)
         logging.debug("Create remote session")
         self.session = remote.remote_login(client, address, port, username,
-                                           password, self.prompt,
-                                           self.linesep, timeout=360,
-                                           status_test_command=self.status_test_command)
+                                           password, prompt,
```

Comment:
In original code, self.prompt/self.linesep/self.status_test_command will be redefined in class NetperfPackage and Netperf, and these definations are the same. Finally, all these params will be only used by remote.remote_login()

Although we have introduce self.prompt/self.linesep/self.status_test_command as params, i think there is no need to use the self.* for these three params when passing them to Netperf, NetperfPackage, remote.remote_login().